docs/faq.rst

```diff
@@ -292,3 +292,17 @@ Some examples follow:
     Please check the `link
     <https://neurostars.org/t/fmriprep-how-to-reuse-longitudinal-and-pre-run-freesurfer/4585/15>`__
     for further information on this approach.
+
+
+How to decrease *fMRIPrep* runtime when working with large datasets?
+--------------------------------------------------------------------
+*fMRIPrep* leverages PyBIDS to produce a layout, which indexes the input BIDS dataset and facilitates file queries.
+Depending on the amount of files and metadata within the BIDS dataset, this process can be time-intensive.
+As of the 20.2.0 release, *fMRIPrep* supports the ``--bids-database-dir`` option, which can be used to pass in an already indexed BIDS layout.
+
+The default *fMRIPrep* layout can be generated by running the following shell command (requires PyBIDS 0.12.1 or greater)::
+
+  pybids layout <path-to-bids-dir> <path-to-save-index> --no-validate \
+  --ignore code --ignore stimuli --ignore sourcedata --ignore models --ignore derivatives --ignore m/^\./
```

Comment:
These should already be ignored by default...

```suggestion
  pybids layout <path-to-bids-dir> <path-to-save-index> --no-validate
```

Reply:
Almost, but `derivatives` is missing from the [pybids defaults](https://github.com/bids-standard/pybids/blob/2fc912f7987b3e56472b6fb15286abee9e4cbe89/bids/layout/validation.py#L33-L34) but present in ours https://github.com/poldracklab/fmriprep/blob/d82bb4d7d652517e32029bc3840e39edc9907f93/fmriprep/config.py#L443-L450

Unless `BIDSLayout` filters those out separately?

Reply:
Derivatives is a special case, and is always ignored. When  you use `--derivatives`, it adds derivatives in a separate stage, after the normal indexing.

Reply:
Cool, I'll update the BIDSLayout initialization for clarity

Reply:
Yeah, I think it was like that because one version of PyBIDS somehow lost the automatic ignores, and we just put them in to avoid having to add a new minimum PyBIDS on all of the dependencies.